diff --git a/src/main/java/org/glizzygladiators/td/entities/Map.java b/src/main/java/org/glizzygladiators/td/entities/Map.java
index 13ff2bc..7d03ecf 100644
--- a/src/main/java/org/glizzygladiators/td/entities/Map.java
+++ b/src/main/java/org/glizzygladiators/td/entities/Map.java
@@ -5,6 +5,23 @@ import org.glizzygladiators.td.entities.towers.Tower;
 public class Map {
 
     private Rectangle[] collisionBound;
+    public int[][] path = {
+        {0, collisionBound[0].y},
+        {795, collisionBound[0].y},
+        {795, collisionBound[2].y},
+        {collisionBound[2].x, collisionBound[2].height},
+        {750, 750},
+        {1000, 1000}
+    };
+    public int[][] offsets = {
+        {1, 0},
+        {0, 1},
+        {-1, 0},
+        {0, 1},
+        {1, 0}
+    };
+    public final int startX;
+    public final int startY;
 
     public Map() {
         collisionBound = new Rectangle[]{
@@ -14,7 +31,8 @@ public class Map {
             new Rectangle(125, 364, 169 - 125, 572 - 364, null),
             new Rectangle(170, 552, 747 - 170, 602 - 552, null)
         };
-
+        startX = path[0][0];
+        startY = path[0][1];
     }
 
     public boolean hasCollisionWithPath(Tower input) {
